kafka-connect-s3/src/main/resources/file-event.avsc

```diff
@@ -4,6 +4,11 @@
   "name": "FileEvent",
   "doc": "This event represents a fileEvent Message",
   "fields": [
+    {
+      "name": "cluster_name",
+      "type": "string",
+      "doc": "The cluster name of the record being written"
```

Comment:
we should make this nullable and add a default value. Also we should introduce a new version in avro kafka config accordingly